### ISSUE TITLE
Bug: uploading to OFF zeros out calories

### ISSUE THREAD

**Issue by rossmeissl** [davidhealey/waistline#475]
Android version: 10
Waistline version: 3.2.2

What I do:
* Open food list
* Scan a barcode for a rare item
* Receive prompt to add to OFF and choose yes
* Fill out fields including both pictures
* Submit
* Scan barcode again

What I expect:
* See the calorie count that I entered

What happens instead:
* Calories are listed as 0

This has happened with every product I've ever uploaded to OFF using Waistline. I'm getting a bit worried that I'm polluting OFF with bad data!

As always thank you for your work on Waistline David.

**Comment by davidhealey** [comment]
Can you give me the barcode number of one of the items?

**Comment by rossmeissl** [comment]
Hi @davidhealey [here's one I uploaded last night](https://world.openfoodfacts.org/product/0898544000323/feta-maplebrook-farm).

**Comment by davidhealey** [comment]
Anyone else having this issue? Could have been something introduced in a recent version.

**Comment by rossmeissl** [comment]
I should clarify that when I said "Fill out fields including both pictures" I mean only the fields that appear on my phone. In my case, I don't care about fat, sat fat, sugars, or salt, so I'm not surprised those are `?`. But curiously I _do_ have fiber enabled on my phone, which I filled out, and which also appears to be missing.

**Comment by EmilJunker** [comment]
I believe this was caused by the introduction of custom nutrients in version 3.2.2. I have opened a [pull request](https://github.com/davidhealey/waistline/pull/476) with some changes that should fix this problem. We should probably push a new release soon since this is a pretty major bug.

> But curiously I do have fiber enabled on my phone, which I filled out, and which also appears to be missing.

Apparently, nutrients with a value of zero were ignored in the upload. This is also fixed now.
